### ISSUE TITLE
CSS Styling

### ISSUE THREAD

**Issue by felix-ulonska** [eea/volto-hero-block#48]
Hello,
Thank you for the awesome plugins!

Where do I find the css styling for the Hero Block. Are there samples how to style the component?

**Comment by avoinea** [comment]
@felix-ulonska Indeed, we didn't manage to provide this add-on with an included minimal style.

You can take a look in our EEA Design System https://github.com/eea/volto-eea-design-system/blob/master/theme/themes/eea/extras/hero.less

**Comment by felix-ulonska** [comment]
@avoinea Thank you for the link to the EEA Design System. How about linking this from the readme.md? To give people a starting point?
